lib/fastlane/plugin/wpmreleasetoolkit/helper/android/android_version_helper.rb

```diff
@@ -305,7 +305,7 @@ def self.get_library_version_from_gradle_config(import_key:)
 
           File.open(gradle_file_path, 'r') do | f |
             text = f.read
-            text.match(/^(?:\w*\.)?#{Regexp.escape(import_key)}\s*=\s*['"](.*?)["']/m)&.captures&.first
+            text.match(/^(\s*)(?:\w*\.)?#{Regexp.escape(import_key)}\s*=\s*['"](.*?)["']/m)&.captures&.last
```

Comment:
🤔 nit: I just realized: we don't need to have a capture group for this whitespace after all, so I think this _should_ work too (while simplifying the regex slightly)?
```suggestion
            text.match(/^\s*(?:\w*\.)?#{Regexp.escape(import_key)}\s*=\s*['"](.*?)["']/m)&.captures&.first
```
(ndb, take it or leave it; since both solutions work, as long as we fix the issue, either are good… just thought I'd mention this as alternative for simplification)

Reply:
Yep you're right I overlooked that! I'll merge it in, thanks @AliSoftware